susemanager-utils/susemanager-sls/src/modules/sumautil.py

```diff
@@ -11,6 +11,7 @@
 import re
 import time
 import salt.utils
+import salt.modules.cmdmod
```

Comment:
@meaksh Just out of curiosity, does this have anything to do with that Python 2.6 compatibility or does this fixes something else?

Reply:
This is actually something else that was spotted by the new GH action to detect non-compatible syntax, as it was causing an `ImportError` when calling the file with `python` directly.

Apparentely this missing `import` was not really causing any issue when executed by Salt and loaded via LazyLoader.

Reply:
Is the `import` also needed if we just compile the code? We don't need to run the code to see syntax errors:

```
➤ podman run -it --rm centos:centos6 /bin/bash
[root@9b505b1e64c4 /]# cat foo.py
results = {k:v for k,v in [(1,1), (2,2)]}
[root@9b505b1e64c4 /]# python -m py_compile foo.py
SyntaxError: ('invalid syntax', ('foo.py', 1, 18, 'results = {k:v for k,v in [(1,1), (2,2)]}\n'))